eu.usdx.UltraStarDeluxe.yaml

```diff
@@ -0,0 +1,194 @@
+- "/lib/*.so"
+- "/lib/*.la"
+- setup_vars_opencv4.sh
+
+cleanup-commands:
+- rm -Rf /var/tmp/fpc
+
+add-extensions:
+  eu.usdx.UltraStarDeluxe.Songs:
+    directory: share/ultrastardx/songs
+    subdirectories: true
+    no-autodownload: true
+
+  eu.usdx.UltraStarDeluxe.Themes:
+    directory: theme-extensions
+    merge-dirs: share/ultrastardx/themes
+    subdirectories: true
+    no-autodownload: true
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: .
+
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.freepascal
+
+build-options:
+  append-path: /usr/lib/sdk/freepascal/bin
+  env:
+   - FPCDIR=/usr/lib/sdk/freepascal/fpcsrc
+
+modules:
+- name: opencv
+  sources:
+  - type: archive
+    url: https://github.com/opencv/opencv/archive/4.7.0.tar.gz
+    sha256: 8df0079cdbe179748a18d44731af62a245a45ebf5085223dc03133954c662973
+  buildsystem: cmake-ninja
+  builddir: true
+  build-options:
+    config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DOPENCV_GENERATE_PKGCONFIG=ON
+    - -DBUILD_TESTS=OFF
+    - -DBUILD_opencv_apps=OFF
+    - -DBUILD_opencv_calib3d=OFF
+    - -DBUILD_opencv_dnn=OFF
+    - -DBUILD_opencv_features2d=OFF
+    - -DBUILD_opencv_flann=OFF
+    - -DBUILD_opencv_gapi=OFF
+    - -DBUILD_opencv_highgui=OFF
+    - -DBUILD_opencv_java_bindings_generator=OFF
+    - -DBUILD_opencv_ml=OFF
+    - -DBUILD_opencv_objdetect=OFF
+    - -DBUILD_opencv_photo=OFF
+    - -DBUILD_opencv_python2=OFF
+    - -DBUILD_opencv_python_bindings_generator=OFF
+    - -DBUILD_opencv_python_tests=OFF
+    - -DBUILD_opencv_stitching=OFF
+    - -DBUILD_opencv_ts=OFF
+    - -DBUILD_opencv_video=OFF
+    - -DCV_TRACE=OFF
+    - -DWITH_PROTOBUF=OFF
+    - -DWITH_FFMPEG=OFF
+    - -DWITH_ADE=OFF
+    - -DWITH_QUIRC=OFF
+    - -DWITH_GTK=OFF
+    - -DWITH_TIFF=OFF
+    - -DWITH_WEBP=OFF
+    - -DWITH_JASPER=OFF
+    - -DWITH_OPENJPEG=OFF
+    - -DWITH_PNG=OFF
+    - -DWITH_OPENEXR=OFF
+    - -DWITH_GDAL=OFF
+    - -DWITH_GDCM=OFF
+    - -DWITH_IMGCODEC_HDR=OFF
+    - -DWITH_IMGCODEC_SUNRASTER=OFF
+    - -DWITH_IMGCODEC_PXM=OFF
+    - -DWITH_IMGCODEC_PFM=OFF
+  cleanup:
+  - /share/opencv4/*cascades
+  - /share/opencv4/valgrind*.supp
+
+- name: projectm
+  sources:
+  - type: archive
+    url: https://github.com/projectM-visualizer/projectm/releases/download/v2.2.1/projectM-2.2.1.tar.gz
+    sha256: 9bbb33c5ba048537e97ea5ba2bd9fef76972c881597599a272b0194ec1d5f2a3
+  - type: patch
+    paths:
+    - patches/projectM-2.2.1.patch
+  buildsystem: autotools
+  rm-configure: true
+  config-opts:
+  - --disable-static
+  post-install:
+  - "f=$(fc-match -f '%{file}' sans) ; sed -i \"/Title Font/s:=.*:= $f:\" ${FLATPAK_DEST}/share/projectM/config.inp"
+  - "f=$(fc-match -f '%{file}' monospace) ; sed -i \"/Menu Font/s:=.*:= $f:\" ${FLATPAK_DEST}/share/projectM/config.inp"
+  cleanup:
+  - /share/projectM/fonts
+  - /bin
+
+- shared-modules/lua5.4/lua-5.4.json
+
+- name: portmidi
+  sources:
```

Comment:
sources is usually last in the module

Reply:
actually it's more intuitive because that's where maintainers expect things to be. It doesn't affect the build.